config/routes.rb

```diff
@@ -1,8 +1,10 @@
 ManualsFrontend::Application.routes.draw do
   with_options(constraints: { prefix: /(guidance|hmrc-internal-manuals)/ }) do |r|
-    r.get '/:prefix/:manual_id', to: 'manuals#index'
-    r.get '/:prefix/:manual_id/updates', to: 'manuals#updates'
-    r.get '/:prefix/:manual_id/:section_id', to: 'manuals#show'
+    scope format: true, constraints: { format: 'html' } do
```

Comment:
I'm not familiar with the `with_options` method but that looks like it's basically doing the same thing as scope and could be replaced and merged with the scope we've got rather than needing nested blocks?